build/tasks/sass.js

```diff
@@ -36,6 +37,16 @@ var iconFiles = "src/icons/**/*.scss";
 
 var assets = ["src/clarity/img/*.png","src/clarity/img/*.svg"];
 
+/*
+ * Adds the current clarity version to generated css files
+ * Depends on this being in the *.scss files: \/* @echo VERSION *\/
+ * NOTE: for this to work we must generate a loud comment -> /*!
+ */
+var env = require('gulp-env');
+var pjson = require('../../package.json'); // Grab the projects package.json w/ version info
```

Comment:
No need to parse the `package.json`, I believe `util.env.version;` is already set when running any npm script.

See [here](https://github.com/vmware/clarity/blob/master/build/tasks/publish.js#L15) for an example of how we do it when publishing to NPM.